tests/unit/test_service.py

```diff
@@ -228,6 +229,38 @@ def _fake_measurement_function():
     pass
 
 
+@pytest.mark.parametrize(
+    "service_config,provided_interfaces",
+    [
+        (
+            "example.serviceconfig",
+            [
+                "ni.measurementlink.measurement.v1.MeasurementService",
+                "ni.measurementlink.measurement.v2.MeasurementService",
+            ],
+        ),
+        ("examples.v1.serviceconfig", ["ni.measurementlink.measurement.v1.MeasurementService"]),
+        ("examples.v2.serviceconfig", ["ni.measurementlink.measurement.v2.MeasurementService"]),
+    ],
+)
+def test__measurement_service__create_measurement_service__service_info_populated_by_serviceconfig(
```

Comment:
The when clause should not say the test creates something that is provided by the given clause. Change the given clause to something like `service_config` because that is the input to `create_measurement`.

I think `matches` is clearer than `populated_by`.

Change `serviceconfig` to `service_config` for consistency.